symfony/panther/1.0/manifest.json

```diff
@@ -4,7 +4,7 @@
         "ENV PANTHER_NO_SANDBOX 1",
         "# Not mandatory, but recommended",
         "ENV PANTHER_CHROME_ARGUMENTS='--disable-dev-shm-usage'",
-        "RUN apt-get install -y --no-install-recommends chromium chromium-driver",
+        "RUN apt-get update && apt-get install -y --no-install-recommends chromium chromium-driver && rm -rf /var/lib/apt/lists/*",
         "",
         "# Firefox and geckodriver",
         "#ARG GECKODRIVER_VERSION=0.34.0",
```

Comment:
Can you also add it for geckodriver, even if it's commented on? https://github.com/symfony/recipes/blob/main/symfony/panther/1.0/manifest.json#L11